lib/smart_proxy_reports/friendly_message.rb

```diff
@@ -27,6 +32,21 @@ def generate_message
     msg
   end
 
+  def assemble_message
+    "Concatenation of #{@result_tree["src"]} into #{@result_tree["dest"]}"
+  end
+
+  def dnf_message
+    package_message
+  end
+
+  def git_message
+    if @module_args_tree["clone"]
```

Comment:
If it's `false` it doesn't necessary mean it wasn't cloned due to missing remote repo, see https://docs.ansible.com/ansible/latest/collections/ansible/builtin/git_module.html#parameter-clone

For test purposes I've set `clone: false` and I've received a misleading message `Check if repository https://github.com/ares/foreman_ansible_test.git exists` although the remote repo exists :)

Reply:
You are right, setting clone to false does not clone the repository, whether or not it is already cloned. I was struggling a bit to understand for what purpose this would be eventually used. But ansible does not notify user in case the remote repo does not exist, so I thought that's what this message is for.

Reply:
See below, yeah let's go with something on the safe side.

Reply:
Let's go with: `Check if clone of https://github.com/ares/foreman_ansible_test.git exists`